generamba/lib/generamba/cli/gen_command.rb

```diff
@@ -39,6 +40,17 @@ def gen(module_name, template_name)
       template = ModuleTemplate.new(template_name)
       code_module = CodeModule.new(module_name, module_description, rambafile, options)
 
+      project = XcodeprojHelper.obtain_project(code_module.xcodeproj_path)
+      module_group_already_exists = XcodeprojHelper.module_with_group_path_already_exists(project, code_module.module_group_path)
+
+      if (module_group_already_exists == true)
+        replace_exists_module = yes?("#{module_name} module already exists. Replace? (yes/no)")
+      
+        if (replace_exists_module == false)
```

Comment:
It's better to post a message in this case, something like `You chose not to overwrite your module. Try using another name.`